exercises/practice/difference-of-squares/.meta/example.cairo

```diff
@@ -0,0 +1,35 @@
+pub fn squareOfSum(number: u64) -> u64 {
+    let mut sum = 0;
```

Comment:
Makes sense. It'd be a good candidate for an [approach](https://exercism.org/docs/building/tracks/approaches) which can be reviewed by a student after they've solved the exercise.

For a practice exercise, the example solution really just needs to pass the CI so it doesn't need to be idiomatic (unless the maintainers want it to be of course). Preferably, the example doesn't take so long that it'd time out on the website if submitted there.

For a concept exercise, I think approaches are still possible, but if they're not, this could be written up as mentoring notes that are shown to the mentor during a mentoring discussion. The mentor will also see the exemplar solution (concept exercise equivalent to a practice exercise's example solution) so that should be as idiomatic as possible for that point in the overall syllabus.

Reply:
I like treating this as an approach :+1:

Reply:
I’ll write something up this weekend.